src/main/java/baritone/cache/CachedWorld.java

```diff
@@ -120,8 +120,7 @@ public final boolean isCached(int blockX, int blockZ) {
                     int regionZ = zoff + playerRegionZ;
                     CachedRegion region = getOrCreateRegion(regionX, regionZ);
                     if (region != null) {
-                        // TODO: 100% verify if this or addAll is faster.
-                        region.getLocationsOf(block).forEach(res::add);
+                        res.addAll(region.getLocationsOf(block));
```

Comment:
Same as CachedRegion, this is actually faster

Reply:
IntelliJ ``// noinspection ...`` when